core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -134,19 +152,16 @@
     font-family: 'Roboto', Arial, sans-serif;
     font-size: 1.0em;
   }
-
```

Comment:
Don't change these -- please leave them as they were.

Reply:
This is a mistake, I didn't want to change it. Sorry about that!